### DISCUSSION TITLE
My GitHub achievements aren't showing

### DISCUSSION THREAD

**Discussion by pecraveiro** [community/community#73194] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I've already cleaned the cookies/cache on my browser, disabled and enabled the profile setting, and it still doesn't show on my profile. However, I've checked my profile on the app, and it shows there.

**Comment by Skillamu** [comment, marked as answer]
@pecraveiro, i had the exact same problem. Go to https://github.com/[your-username]?tab=achievements. Click on any achievement, click on the eye icon to "Hide from profile" and then click on the eye icon again to "Show on profile".

Weird bug, but this solved the problem in my case.

**Reply by pecraveiro** [reply]
Wow, thank you so much! That fixed it for me too! I hadn't noticed that option anywhere else.

**Reply by abthahiahmed** [reply]
Thank you so much. This bug is still there. But thanks to you for the solution.

**Reply by santoshpanda1995** [reply]
The bug is not solved yet, But thanks for your answer, which fixed it for me.

**Reply by Zineeddine-Abd** [reply]
**Thank you sir**, weird bug but it's solved now!

**Reply by amit77t** [reply]
Thank you so much, it works

**Reply by jacq-tandang** [reply]
I get a 404 page if I do this.
Worked after checking "Include private contributions on my profile" in settings

**Reply by Devashermale** [reply]
The bug is not solved yet, But thanks for your answer, which fixed it for me.

**Comment by moha999DJ** [comment]
The solution is very simple 
1-first go to the setting
2-then "Contributions & activity"
3-choose the second choise
"Include private contributions on my profile"
4-enter update preferences and congratulations 🎊

**Comment by bayramtturgutt** [comment]
@abthahiahmed, @santoshpanda1995, it is worth checking whether it is hidden in the profile

![image](https://github.com/user-attachments/assets/d3ec37ef-6575-4176-b819-6e139ff66779)

**Reply by abthahiahmed** [reply]
Please visit https://github.com/yourusername/?tab=achievements

Then you can see your achievements there.

**Comment by Sakith2820** [comment]
Please visit https://github.com/yourusername/?tab=achievements

from this I'm landing on the 404 error page

**Reply by RAJ8664** [reply]
Replace "yourusername" -->  with your github username in the given link.

**Reply by Elmbark** [reply]
The solution is very simple
1-first go to the setting
2-then "Contributions & activity"
3-choose the second choise
"Include private contributions on my profile"
4-enter update preferences and congratulations 🎊

**Reply by paulcodes777** [reply]
The complete solution listed below
!. Go to settings

2. On public profile scroll down until you see "Contributions & activity"

3. Choose "Include private contributions on my profile"

4. Update prefrences 


5. Visit this link         https://github.com/yourusername/?tab=achievements       You will replace 'yourusername' with your actual github username

6. You will then see your achievements 

7. Click on the achievement 

8. On the top left you will see an eyeball

9. Click on the eyeball and make sure it says display on profile

10. Now when you refresh your github account you will be able to see your achievement 


This ended up being the solution that helped me (and hopefully others)— feel free to mark it as the answer

**Reply by Devashermale** [reply]
The solution is very simple
1-first go to the setting
2-then "Contributions & activity"
3-choose the second choise
"Include private contributions on my profile"
4-enter update preferences and congratulations 🎊

**Comment by paulcodes777** [comment]
The complete solution listed below

1. !. Go to settings

2. On public profile scroll down until you see "Contributions & activity"
 
3. Choose "Include private contributions on my profile"
 
4. Update prefrences
 
5.  Visit this link https://github.com/yourusername/?tab=achievements You will replace 'yourusername' with your actual github username
 
6.  You will then see your achievements
 
7.  Click on the achievement
 
8.  On the top left you will see an eyeball
 
9.  Click on the eyeball and make sure it says display on profile
 
10. Now when you refresh your github account you will be able to see your achievement

This ended up being the solution that helped me (and hopefully others)— feel free to mark it as the answer

**Comment by RubensZimbres** [comment]
Same problem, as of March 27th, 2026. I have 7 achievements, only 2 are showing.

**Comment by ashvardanian** [comment]
Same issue here! Many achievements, but only two are showing.

**Comment by JKamsker** [comment]
Same

**Comment by aitezazdev** [comment]
i have many pr but still not showing any achievement, anyone know how to resolve?

**Comment by atiqullah-ahmadzai** [comment]
Same, no achievement tab or badges on my profile at all
- This throughs 404: https://github.com/atiqullah-ahmadzai?tab=achievements
- "Include private contributions on my profile" is checked
- "Show Achievements on my profile" is checked
